Exiled.API/Features/Round.cs

```diff
@@ -79,6 +79,11 @@ public static bool IsLobbyLocked
         /// </summary>
         public static int ChangedIntoZombies => RoundSummary.changed_into_zombies;
 
+        /// <summary>
+        /// Gets the round.
```

Comment:
This should be changed to be more descriptive, and/or to actually describe what it actually is